aim/tools/cli/debug/hashtree.py

```diff
@@ -65,6 +65,35 @@ def dump(ctx, tenant, flavor):
             click.echo('%s not found for tenant %s' % (search.__name__, t))
 
 
+@hashtree.command(name='dump-amt')
```

Comment:
This is a real nit-pick, but can we change the command to something a bit more descriptive? Maybe "count-tree-nodes"?